web/components/buttons/follow-button.tsx

```diff
@@ -61,8 +61,11 @@ export function UserFollowButton(props: { userId: string }) {
   )
 }
 
-export function MiniUserFollowButton(props: { userId: string }) {
-  const { userId } = props
+export function MiniUserFollowButton(props: {
```

Comment:
The formatting for this is messed up when you click follow:
https://www.loom.com/share/94fcfc8d7033410d856e957d938a4221